dwitter/static/main.css

```diff
@@ -274,13 +274,11 @@ iframe {
 }
 
 .head-account-info .collapsible {
-  opacity: 0;
   position: absolute;
   border: 1px solid #ddd;
   right: 1em;
-  margin-top: -4px;
-  transition: all 0.05s;
```

Comment:
I think it would be nice to keep this transition. Would changing it to `transition: opacity 0.05s` work?